docs/pages/combobox/index.mdx

```diff
@@ -1,26 +1,37 @@
-# Combobox
-
+# ComboBox
 
+An accessible ComboBox with single, multi & accessible select with typeahead &
+autocomplete support.
 
 ## Import
 
-
-
 ```js
-import {
-  MySelect
-} from "@chakra-components/core";
+import { ComboBox } from "@chakra-components/core";
+```
 
+## Usage
 
+Here's a basic usage of the Select component.
 
+```jsx
+<ComboBox>
+  <ComboBoxLeftElement color="gray.300" fontSize="1.2em" children="$" />
+  <ComboBoxRightElement
+    children={<Icon name="chevron-down" fontSize="1.5rem" />}
+  />
+  <ComboBoxClearElement />
```

Comment:
TODO: I need to plugin clear here.

Reply:
You need plugin clear? I did not understand your motive. Explanation?

Reply:
It was a TODO for me to fix this.
1. Close Icon when we type something
2. Preloader when a async search happens.
3. No icon when there is no text. I will try to achieve this in my async branch.